portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,102 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+@import url('https://fonts.googleapis.com/css2?family=Days+One&family=Lora:ital@0;1&display=swap');
+
+body {
+  font-family: 'Lora', serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: normal;
+  margin-left: 20%;
+  margin-right: 20%;
+}
+
+p {
+  line-height: 29px;
+}
+
+.section {
+  margin: 8% 0 8% 0;
+}
+
+h1 {
+  font-size: 64px;
+}
+
+h2 {
+  font-size: 48px;
+  padding-top: 30px;
+}
+
+h1, h2 {
+  font-family: 'Days One', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+}
+
+button {
+  background-color: transparent;
+  border: none;
+}
+
+button:hover {
+  cursor: pointer;
+}
+
+a {
+  color: black;
+  text-decoration: none;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#main-nav {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  margin-top: 6%;
+  width: 100%;
 }
+
+#profile-img {
+  margin-right: 7%;
+  width: 364px;
+}
+
+nav > ul {
+  list-style-type: none;
+  flex-grow: 1;
+  margin-right: 7%;
+  padding-left: 0;
+}
+
+nav > ul > li {
+  border-bottom: 1px solid black;
+  padding: 30px 0 15px 0;
+  width: 100%;
+}
+
+nav > ul > li > a {
+  font-size: 24px;
+}
+
+.button-panel {
+  margin: 50px 0 50px 0;
```

Comment:
I believe you can simplify this to be 50px 0

since top/bottom and left/right values are identical, same thing at the bottom with .button-content > p padding